src/components/app-map/map-layer.vue

```diff
@@ -16,7 +16,7 @@ export default {
   date() {
```

Comment:
I think I've typed this wrongly a million times as well 😄. Probably should've been `data`